pgo/cmd/create.go

```diff
@@ -241,7 +256,9 @@ func init() {
 	createClusterCmd.Flags().StringVarP(&ReplicaStorageConfig, "replica-storage-config", "", "", "The name of a Storage config in pgo.yaml to use for the cluster replica storage.")
 	createClusterCmd.Flags().StringVarP(&NodeLabel, "node-label", "", "", "The node label (key=value) to use in placing the primary database. If not set, any node is used.")
 	createClusterCmd.Flags().StringVarP(&ServiceType, "service-type", "", "", "The Service type to use for the PostgreSQL cluster. If not set, the pgo.yaml default will be used.")
-	createClusterCmd.Flags().StringVarP(&Password, "password", "w", "", "The password to use for initial database users.")
+	createClusterCmd.Flags().StringVarP(&PasswordUser, "user-password", "", "", "The password to use for standard user account created during cluster initialization.")
```

Comment:
This is the one I feel the strongest about. We should leave `password` (it is already tailored to the flags one expects with PostgreSQL`) and make it clear it's for the user password

Reply:
Changed the CLI flag back to `password` for the user password.